src/main/java/frc/robot/subsystems/intake/IntakeIOReal.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfigurator;
+import com.ctre.phoenix6.controls.DutyCycleOut;
+import com.ctre.phoenix6.controls.PositionVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import frc.robot.Constants;
+import frc.robot.Ports;
+
+public class IntakeIOReal implements IntakeIO {
+
+    private final CANSparkMax spinMotor =
+            new CANSparkMax(Ports.Intake.INTAKE_FRONT_MOTOR, CANSparkLowLevel.MotorType.kBrushless);
+    private final CANSparkMax centerMotor =
+            new CANSparkMax(
+                    Ports.Intake.INTAKE_CENTER_MOTOR, CANSparkLowLevel.MotorType.kBrushless);
+    private final TalonFX angleMotor = new TalonFX(21);
+    private final TalonFXConfigurator angleConfigurator;
+    private final DutyCycleOut dutyCycleRequest = new DutyCycleOut(0);
```

Comment:
Unless you want to [run the intake at the maximum speed possible](https://pro.docs.ctr-electronics.com/en/latest/docs/api-reference/device-specific/talonfx/talonfx-control-intro.html#dutycycle), you can use voltage control, or better, torque current control. With these you also don't need to set voltage compensation.

Reply:
was not used

Reply:
NEVERmINED

Reply:
4c87cd31b16f775ea2977caa7b146b3dce728aeb

Reply:
4c87cd31b16f775ea2977caa7b146b3dce728aeb